packages/atomic/src/components/facets-v1/facet-value-label-highlight/facet-value-label-highlight.tsx

```diff
@@ -0,0 +1,16 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {FacetValueLabelHighlightProps} from '../facet-common';
+import {highlightSearchResult} from '../facet-search/facet-search-utils';
+
+export const FacetValueLabelHighlight: FunctionalComponent<FacetValueLabelHighlightProps> = (
+  props
+) => {
+  return (
+    <span
+      title={props.displayValue}
+      part="value-label"
```

Comment:
For the atomic-rating-facet, make sure to keep the document parts up to date. I don't think it uses the `value-label` part anymore. Perhaps create a new part for value-rating?